connection/connection.go

```diff
@@ -17,8 +17,6 @@ package connection
 import (
 	"crypto/tls"
 	"crypto/x509"
```

Comment:
Note to self. We probably can get the rid of these dependencies too, `loadServerTrust` should not be needed neither.